### ISSUE TITLE
Missing accessibility labels for Android Tablet

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#7906]
## Description 
Android Tablet doesn't have accessibility labels for Rewards and Shields

## Steps to reproduce 

   1. Long press on Rewards/Shields
   2. No accessibility labels are shown

## Actual result 
Missing accessibility labels for Android Tablet

## Expected result
Should show accessibility labels 

## Issue reproduces how often 
Easy

## Issue happens on 
   - Current Play Store version? No
   - Beta build? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device (Phone, Tablet, Phablet): Samsung Tab A Tablet
   - Android version: 9.0
   
## Brave version
1.5.57

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome?  NA

### Additional information

Follow up to  #7215

**Comment by srirambv** [comment]
Verification passed on Samsung Tab A with Android 9.0 running 1.6.24 
![image](https://user-images.githubusercontent.com/17010094/74433368-477a4100-4e86-11ea-8a3f-414021038b2a.png)
![image](https://user-images.githubusercontent.com/17010094/74433375-4a753180-4e86-11ea-99f3-47f3bc2f6d18.png)
